tests/virtualization/xen/hotplugging.pm

```diff
@@ -35,9 +35,11 @@ sub run_test {
                 record_soft_failure 'bsc#1168124 Bridge network interface hotplugging has to be performed at the beginning.';
                 $self->{test_results}->{$guest}->{"bsc#1168124 Bridge network interface hotplugging has to be performed at the beginning"}->{status} = 'SOFTFAILED';
                 $persistent_config_option = '--persistent' if ($sles_running_version eq '11' && $sles_running_sp eq '4');
+                script_run "brctl addbr br0; ip link set dev br0 up", 60 if ($sles_running_version eq '11' && $sles_running_sp eq '4');
```

Comment:
Can you explain what br0 is for? I am curious why there is no "brctl addif xx" to bind it to a real NIC? BTW is it better to check whether br0 exists before creation?

Reply:
SLE 11-SP4 on OSD does not have br0 after installation. And network interface hotplugging needs br0. So it needs to be created manually in advance. I do not think checking br0's existence is necessary, because 1) this step only applies to 11-SP4 host. Checking br0's existence seems complicates things here.  2) As I said above, the root cause is untypical 11-SP4 virtualization host installation which is very rare case. So it does not deserve more complexity. I already confined the code change to the minimum 3) We actually do not need a fully functioning br0, so we do not care too much about it. We just need it there anyway, that is enough.

Reply:
Ok. So in short, current sle11sp4 host does not have br0, so directly creating it can solve the problem already with least code lines. I agree. Then what if in future the autoyast for 11p4 host installation changes, and adds br0? Who knows. Yes, you can say let's do it then. This can be a way. 

However from coding habit perspective, I would say making it work "safely" for more circumstances will benefit us for longer time. This is the point I want to deliver actually. Not so strictly to this specific code change.

Of course, if you insist, I agree to merge.

Reply:
This is not the test scenario thing or different product flavor we need to cover. Actually we do workaround here. So we just make it work, not to solve a problem completely. And I also can not get on with designing a not-very-useful all-round utility judging by the situation here. Your point is right, but not very applicable to this case. You can help merge if there is no other defects directly related.